src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -749,8 +750,8 @@ public KeyToVectorMappingEstimator(IHostEnvironment env, params KeyToVectorMappi
         {
         }
 
-        public KeyToVectorMappingEstimator(IHostEnvironment env, string inputColumn, string outputColumn = null, bool bag = Defaults.Bag)
-            : this(env, new KeyToVectorMappingTransformer(env, new KeyToVectorMappingTransformer.ColumnInfo(inputColumn, outputColumn ?? inputColumn, bag)))
+        public KeyToVectorMappingEstimator(IHostEnvironment env, string outputColumnName, string sourceColumnName = null, bool bag = Defaults.Bag)
```

Comment:
>KeyToVectorMappingEstimator [](start = 15, length = 27)

Not related to your change. This class is missing xml documentation over all.

Reply:
i know, but the scope of work was too large, and didn;t want to expand further. ...

---
In reply to: [251641257](https://github.com/dotnet/machinelearning/pull/2239#discussion_r251641257) [](ancestors = 251641257)